packages/mermaid/src/diagrams/sequence/sequenceDb.js

```diff
@@ -530,7 +532,7 @@ export const apply = function (param) {
             lastDestroyed = undefined;
           }
         }
-        addSignal(param.from, param.to, param.msg, param.signalType);
+        addSignal(param.from, param.to, param.msg, param.signalType, param.activate);
```

Comment:
I feel like we need to add a JSDoc comment for the `activate` value somewhere (although no functions in this file seem to have any comments).

Does it make sense to do something like:

```js
/**
 * @typedef {Object} AddMessageParams A message from one actor to another.
 * //TODO: add missing types
 * @property {"addMessage"} type - Set to `"addMessage"` if this is an `AddMessageParams`.
 * @property {boolean} [activate] - If `true`, this signal starts an activation.
 */

/**
 * @param {Object | AddMessageParams} param - Object of parameters.
 */
export const apply = function (param) {
```

We don't need to add types/documentation for everything (especially since this is still a JavaScript file, not TypeScript). Just having docs for the new `activate` param that you added is good enough.